server/auth_test.go

```diff
@@ -44,7 +44,7 @@ var configAuth2 = server.Configuration{
 func TestMissingAuthToken(t *testing.T) {
 	helpers.AssertAPIRequest(t, nil, &configAuth, &helpers.APIRequest{
 		Method:   http.MethodGet,
-		Endpoint: server.OrganizationsEndpoint,
+		Endpoint: "organizations/1/clusters",
```

Comment:
Sorry, forgot to tell in previous review, you can use `server.ClustersForOrganizationEndpoint` and add `EndpointArgs: []interface{}{1}` like here https://github.com/RedHatInsights/insights-results-aggregator/blob/master/server/server_test.go#L121